spec/controllers/api/v4/blood_sugars_controller_spec.rb

```diff
@@ -103,6 +103,19 @@ def create_record_list(n, options = {})
         end
       end
 
+      context 'hba1c blood sugars' do
+        it 'successfully records hba1c blood sugars' do
+          blood_sugar = build_blood_sugar_payload(build(:blood_sugar, blood_sugar_type: :hba1c))
+
+          post(:sync_from_user, params: { blood_sugars: [blood_sugar] }, as: :json)
+          errors = JSON(response.body)['errors']
+
+          expect(response).to have_http_status(200)
+          expect(errors).to eq([])
+          expect(BloodSugar.where(id: blood_sugar['id']).count).to eq 1
```

Comment:
Let's assert the right value got in too, since we're dealing with a lot of types.